data-raw/sao-paulo-activity-data.R

```diff
@@ -19,5 +20,27 @@ sao_paulo_activity_df_2 = OD_SP_2017 %>%
 sao_paulo_activity_df_20 = OD_SP_2017 %>%
   filter(ID_PESS %in% sample(people, 20))
```

Comment:
I think it may be better to use something like `filter(ID_PESS %in% people[500:520])` instead of a random sample. Every time we select the sample we will have to update the `case_when` classification of modes to match the A/B street. The same for the df with 2 people.

Reply:
Good point. Or we update the `case_when` to take any input...